packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-basic-sign-in/main/nodejs/integrationsteps.md

```diff
@@ -45,13 +45,13 @@ When the user correctly supplies their password, `IdxStatus` equals `IdxStatus.S
 
    // Handle other statuses
 }
-```
+   ```
 
-#### Other authentication statuses
+   #### Other authentication statuses
 
-Handle other returned `IdxStatus` cases if the user didn't sign in successfully or there are other factors to verify. For example:
+   Handle other `IdxStatus` cases as follows:
```

Comment:
calling out that this is "returned" is helpful. Altho not worded that well in the original - they are providing examples of other "returned `IdxStatus` cases"